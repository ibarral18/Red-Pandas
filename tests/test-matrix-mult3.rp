def matrix int [2][2] mult(matrix int [2][3] x, matrix int [3][2] y)
{
  return x * y;
}

def int main()
{
    int i;
    int j;
    matrix int [2][3] x;
    matrix int [3][2] y;
    matrix int [2][2] prod;

    

	x = [[1,2,3],
         [4,5,6]];

    y = [[9,8],
         [6,5],
         [3,2]];

    prod = mult(x,y);

    for(j = 0; j < prod.row; j = j + 1 ){
		  for(i = 0; i < prod.col; i = i + 1){
		  	print(prod[j][i]);
		  } 
      printStr("");
	  }

    
  return 0;
}