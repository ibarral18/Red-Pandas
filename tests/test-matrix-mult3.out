30	24	
84	69	
